resources/views/controller.blade.php

```diff
@@ -233,6 +236,62 @@ public function destroy(Destroy{{ $modelBaseName }} $request, {{ $modelBaseName
         return redirect()->back();
     }
 
+
+    @if(!$withoutBulk)/**
+    * Remove the specified resources from storage.
+    *
+    * {{'@'}}param  Destroy{{ $modelBaseName }} $request
+    * @return Response|bool
+    * @throws \Exception
+    */
+    public function bulkDestroy(Destroy{{ $modelBaseName }} $request) : Response
+    {
+        $bulkChunked = array_chunk($request->data['ids'], 1000, true);
+
+        DB::transaction(function () use ($bulkChunked){
+            collect($bulkChunked)->each(function($bulkItem){
+            {{ $modelBaseName }}::whereIn('id', $bulkItem)->delete();
+
+                // TODO your code goes here
+            });
+        });
+
+        if ($request->ajax()) {
+            return response(['message' => trans('brackets/admin-ui::admin.operation.succeeded')]);
+        }
+
+        return redirect()->back();
+    }
+
+    /**
+    * Get the resources.
+    *
+    * {{'@'}}param  Index{{ $modelBaseName }} $request
+    * {{'@'}}return Response|array
+    */
+    public function getAll(Index{{ $modelBaseName }} $request)
```

Comment:
@RichardDominik What is this endpoint for?

Reply:
@palypster it is used to select all items when user clicks on select all option

Reply:
Shouldn't we use standard controller's `index` method rather? Otherwise developer needs to keep this two methods synced. Also whole use of `processRequestAndGet` doesn't make sense at all if we use separate method.